### DISCUSSION TITLE
How to configure the dim sizes and learning rates?

### DISCUSSION THREAD

**Discussion by vinay-k12** [gretelai/gretel-synthetics#141] [Q&A]
I'm trying to customize the Gen/Des dim sizes, learning rates and other params in the models. I'm using the same API as SDV. 

import pandas as pd
from gretel_synthetics.actgan import ACTGAN
model = ACTGAN(
    verbose=True,
    binary_encoder_cutoff=10, 
    auto_transform_datetimes=True,
    epochs=100,
    epoch_callback=epoch_tracker.add,
    generator_dim=(1024, 1024),
   discriminator_dim=(1024, 1024),
   generator_lr=0.0000001, generator_decay=1e-05, discriminator_lr=0.000001, discriminator_decay=1e-06,  
)

This worked. Does this mean I can use the same API dictionary as SDV? And can I load models like in SDV (conditions, seeing etc)?

**Comment by johntmyers** [comment]
Yes most of the functionality of CTGAN is supported with ACTGAN, you can see the supported parameter list is just about the same, plus the ones we added:

https://synthetics.docs.gretel.ai/en/stable/models/actgan.html

**Reply by vinay-k12** [reply]
That's great. But I notice that there is not enough info about 'generate' like 'Conditional' or 'Seeding with columns' etc. Are there any references for the same?

**Reply by johntmyers** [reply]
It should support the same methods as CTGAN. So methods like `sample_remaining_columns` are all supported.

**Reply by vinay-k12** [reply]
I tried doing conditional sampling (Column B === 'Value'), but it didn't work.

**Reply by johntmyers** [reply]
Can you be more specific on what did not work? Did you receive errors that you can share?
